wfl/Database/MongoDBManager.cs

```diff
@@ -26,6 +26,19 @@ public MongoDBManager()
             await collection.InsertOneAsync(r);
         }
 
+        public async Task<bool> CheckRestaurantIdAsync(string id)
```

Comment:
If I attempt to get by an invalid Id, it doesn't return null - it throws an error saying `sequence contains no elements`.